### PR TITLE
Replace dependency to PyPlot by CairoPlot. And more ...

### PR DESCRIPTION
**Pull request by Azzaare**
This build upon @rmsrosa work on CairoPlot.jl

The Manifest currently point to my fork of CairoPlot.jl, but as soon as @rmsrosa merge my changes, we can make it point back to the original repo.

No more Python :p

### CODE REVIEW

**Comment by Azzaare** [comment]
@rmsrosa @tk3369 
I actually added more changes, and I want them to be merged quickly ... sorry

**Comment by Azzaare** [comment]
Here is the comprehensive list of changes:
- Update Manifest format to 1.7 (so yes we would need 1.7 on the VPS which is fine)
- Switch the Plot engine to Cairo thanks to CairoPlot.jl
- Simplify `jc juliacon2021` to `jc 2021`
- Most important, switch to the real current day instead of a fake for `jc today` and `jc now`

File: src/command/ig.jl [hunk 1]
```diff
@@ -686,23 +686,16 @@ end
...
     from_date, to_date = extrema(dates)
     last_price_str = format_amount(last(values))
-    p = plot(dates, values,
-        title = "$symbol Historical Prices ($from_date to $to_date)\nLast price: $last_price_str",
-        linewidth = 2,
-        size = (width, height),
-        legend = nothing,
-        tickfontsize = 12,
-    )
+    c = crplot(dates, values; xticks = 5, yticks = 10, title="$symbol Historical Prices ($from_date to $to_date)\nLast price: $last_price_str")
```
Comment (rmsrosa) [inline src/command/ig.jl:691]: You can change the `width` and the `height` too, which are the respective keywords in `crplot` (don't have `size` yet). I will add more keywords later.

Comment (Azzaare) [reply src/command/ig.jl:691]: We can make changes to that in the following days, I think.

Comment (rmsrosa) [reply src/command/ig.jl:691]: yeah, it was mostly to let you know.

**Review by rmsrosa — APPROVED** [review]
Looks good!

**Pull request merged**